### DISCUSSION TITLE
Comment puis-je contribuer à GitHub en tant que testeur QA junior ?<Your question or feedback about the GitHub Discussions product>

### DISCUSSION THREAD

**Discussion by ghost** [community/community#83925] [Discussions]
### Select Topic Area

Question

### Body

Tout est dans le titre ;-)

**Comment by queenofcorgis** [comment]
Hi @SophieSam , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing any posts in languages other than English for now, including this one.

Please note for future discussions: You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. Good luck!
